Sources/EventOffline.swift

```diff
@@ -63,7 +63,11 @@ extension SentryClient {
 	/// Path of directory to which events will be saved in offline mode
 	private func directory() -> String? {
 		guard let documentsPath = NSSearchPathForDirectoriesInDomains(.DocumentDirectory, .UserDomainMask, true).first else { return nil }
-		let directory: String = "\(directoryNamePrefix)\(dsn.serverURL.absoluteString.hashValue)"
+        #if swift(>=2.3)
+		let directory: String = "\(directoryNamePrefix)\(dsn.serverURL.absoluteString!.hashValue)"
```

Comment:
I think `guard let serverURLString = dsn.serverURL.absoluteString else { return nil }` would work as well. I wasn't afraid of the force unwrap because if we have a dsn then the `!` won't fail but we can change it.